### ISSUE TITLE
Add complex portal 

### ISSUE THREAD

**Issue by fehrhart** [PathVisio/pathvisio#159]
...as a choice in the database selection drop down menu for complex annotations in PathVisio. 
https://www.ebi.ac.uk/complexportal/home

**Comment by DeniseSl22** [comment]
This should be integrated, when the new BridgeDb libraries are included (given that the complex portal database has been added to datasources.txt ;) ). @mkutmon @egonw : any updates on including latest bridgedb libraries in PV?

**Comment by egonw** [comment]
See https://github.com/PathVisio/pathvisio/pulls

**Comment by mkutmon** [comment]
yes, I need to test the new libraries - there were some issues with merging
